source/global.txt

```diff
@@ -12,109 +12,26 @@ Global Deployment Paradigm
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-To improve performance for critical operations,
-all {+service+} deployments allow you to deploy data
-close to your users' geographic location, which reduces latency.
-However, deploying to a single region sacrifices high availability if
-there is a regional outage, and using a multi-region deployment requires
-region specificity. Additionally, for compliance with data residency
-laws, you can partition data in global deployments to reside in
-specific regions, ensuring adherence to local
-regulations. You can configure a global deployment for
-high availability, data sovereignty, low latency for a global audience,
-*and* avoid vendor lock-in.
-
-A global deployment may be best for you if you have
-the following requirements:
-
-- You want to use a single cloud provider *or* multiple cloud providers
-- You want to deploy to more than one region for high availability
-- Your application requires low latency *and* has global users, since
-  global deployments serve content from the region closest to the user
-  across zones that you specify.
-
-For example, for an application deployed with |aws| with users
-located globally, you can use a template to deploy a global deployment
-to six :guilabel:`Highest Priority` regions, as shown in the following
-image:
-
-.. image:: /includes/images/global-template.png
-   :alt: "An image showing latency per geographic area for a global cluster with six regions."
-   :width: 600px
-   :align: center
-
-The previous template deploys at least one shard per region for the 
-following regions:
-
-- N. Virginia (``us-east-1``)
-- Oregon (``us-west-2``)
-- Sao Paulo (``sa-east-1``)
-- Frankfurt (``eu-central-1``)
-- Singapore (``ap-southeast-1``)
-- Tokyo (``ap-northeast-1``)
-
-{+service+} uses the geographic location of your zones to serve content
-and performs write from the region nearest a user. You can also
-add additional nodes in other regions to each global {+cluster+} zone.
-This ensures low latency from any location globally, while offering
-high availability if there's a regional outage that affects the primary
-shard.
-
-If your application requires low latency for users in specific regions,
-consider a :ref:`arch-center-paradigms-multi-region`.
-
-If your application requires low latency for users in specific regions,
-high availability, and deployment across multiple cloud providers,
-consider a :ref:`arch-center-paradigms-multi-cloud`.
-
-If your application requires low latency but doesn't require
-cross-region high
-availability, consider a :ref:`arch-center-paradigms-single`.
+- You need a single global connection string 
+- You need to perform global aggregations across all customers
+- You need the ability to read/write for all customers from everywhere 
+  in one logical cluster, while also having regional reads/writes
```

Comment:
np: these are full sentences, so we can add periods at the end of each list item
```suggestion
- You need a single global connection string. 
- You need to perform global aggregations across all customers.
- You need the ability to read/write for all customers from everywhere 
  in one logical cluster, while also having regional reads/writes.
```